docs-src/python/how-to-listen-to-heartbeats-in-python.md

```diff
@@ -0,0 +1,29 @@
+---
+id: how-to-listen-to-heartbeats-in-python
+title: How to heartbeat an Activity in a test in Python
+sidebar_label: Heartbeat an Activity in a test
+description: To listen to a Heartbeat in a test, use the on_heartbeat property.
+tags:
+  - developer-guide
+  - sdk
+  - python
+  - testing
+---
+
+To test a Heartbeat in an Activity, use the [`on_heartbeat()`](https://python.temporal.io/temporalio.testing.ActivityEnvironment.html#on_heartbeat) property of the [`ActivityEnvironment`](https://python.temporal.io/temporalio.testing.ActivityEnvironment.html) class. This property sets a custom function that will be called every time the `activity.heartbeat()` function is called within the Activity.
+
+```python
+@activity.defn
+async def activity_with_heartbeats(param: str):
+    activity.heartbeat(f"param: {param}")
```

Comment:
```suggestion
    activity.heartbeat("param: {param}")
```
Is the `f` in the argument some weird Python casting? A stray character? Missing an `=`? Inquiring minds 'nd whatnot...

Reply:
TIL something. Been a while since I coded in Python, obvs.